packages/contracts-bedrock/test/L2/GasPriceOracle.t.sol

```diff
@@ -93,3 +102,96 @@ contract GasPriceOracle_Test is CommonTest {
         assertEq(returndata, hex"");
     }
 }
+
+contract GasPriceOracleEcotone_Test is GasPriceOracle_Test {
+    /// @dev Sets up the test suite.
+    function setUp() public virtual override {
+        super.setUp();
+
+        // Define the function signature
```

Comment:
You can remove a bunch of code here by using the library function that you defined